neuralmonkey/tests/test_vocabulary.py

```diff
@@ -49,6 +49,7 @@ def test_there_and_back_self(self):
     def test_min_freq(self):
 
         vocabulary = Vocabulary()
+        vocabulary.correct_counts = True
```

Comment:
Jo takhle to myslíš. To tam podle mě nemá co dělat to zapínátko. Tam to má spadnout.